AI/BattleAI/BattleEvaluator.cpp

```diff
@@ -769,10 +823,14 @@ bool BattleEvaluator::attemptCastingSpell(const CStack * activeStack)
 
 #if BATTLE_TRACE_LEVEL >= 1
 						logAi->trace(
-							"Spell affects %s (%d), dps: %2f",
+							"Spell %s to %d affects %s (%d), dps: %2f oldHealth: %d newHealth: %d", 
+							ps.spell->getNameTranslated(),
+							ps.dest.at(0).hexValue.hex,
```

Comment:
Will Armageddon have ps.dest.at(0).hexValue.hex?

Reply:
No idea. :o

Reply:
Just to ensure .at(0) will not crash

Reply:
Okay, I check ps.dest.empty() and in this case give another message without a location.